scout/server/blueprints/cases/templates/cases/case.html

```diff
@@ -280,6 +284,21 @@
   </div>
 {% endmacro %}
 
+{% macro roh_panel() %}
+<div class="panel panel-default" id="rohPanel">
+  <div data-toggle='tooltip' class="panel-heading" title="down format" >
+    Ideogram, UPD, ROH
+  </div>
+  <div class="panel-body">
+    <svg id="roh_svg" width="1900" height="850"></canvas> 
+  </div>
+  <div class="panel-footer">
+    <button class="btn btn-default form-control" data-toggle="modal" data-target="#edit-synopsis">Edit</button>
```

Comment:
Edit synopsis? Nah, that should only be on the synopsis_panel!

Reply:
Fixed! (0eb3edc)